physics/module_sf_mynn.F90

```diff
@@ -728,6 +796,7 @@ SUBROUTINE SFCLAY1D_mynn(                                      &
          GOVRTH(I)=G/TH1D(I)
       ENDDO
 
+      !tgs - should QFX and HFX be separate for land, ice and water?
```

Comment:
If we go back to using the flux calculations, then yes they should be broken down into components.